packages/bigquery/sodasql/dialects/bigquery_dialect.py

```diff
@@ -70,10 +70,13 @@ def __query_table(self, table_name):
         """
         return query
 
-    def sql_test_connection(self, dataset_id) -> Union[Exception, bool]:
+    def sql_test_connection(self) -> Union[Exception, bool]:
         logging.info(f'Listing tables to check connection')
+        project_id = self.account_info_dict['project_id']
+        dataset_id = f'{project_id}.{self.dataset_name}'
         try:
             tables = self.client.list_tables(dataset_id)
+            logging.info(f'dataset_id = {dataset_id}')
```

Comment:
Maybe you want the log before list_tables call or maybe even before try, because it won't print if list_tables raises an exception